geowebcache/wmts/src/main/java/org/geowebcache/service/wmts/WMTSGetCapabilities.java

```diff
@@ -750,10 +750,11 @@ private void layerResourceUrls(
         }
         if (layer instanceof TileJSONProvider) {
             TileJSONProvider provider = (TileJSONProvider) layer;
-            String format = "json";
             if (provider.supportsTileJSON()) {
-                String template = baseTemplate + "/{style}/tilejson?format=" + format;
-                layerResourceUrlsGen(xml, format, "TileJSON", template);
+                for (String format : mimeFormats) {
+                    String template = baseTemplate + "/{style}/tilejson?format=" + format;
```

Comment:
Hmm.. I would have expected to see the format in the URL. The &format parmaeter normally controls the format of what is returned (tilejson is always json).

Reply:
Fixed